roles/change-paths/tasks/main.yml

```diff
@@ -109,12 +65,11 @@
     - "{{ protocols }}"
   notify: Clear site cache
 
+# Note: I don't think this will ever get run, because we've already replaced
```

Comment:
I created https://stanfordits.atlassian.net/browse/SITES-596 for this and the other one below.